elements/urth-core-dataframe/urth-core-dataframe.html

```diff
@@ -138,6 +237,18 @@
 
         ready: function() {
             console.debug("urth-core-dataframe ready");
+            this._contentObserver = new MutationObserver( this._handleQueryChildrenChanged.bind(this));
```

Comment:
Can you not use [`observeNodes`](https://www.polymer-project.org/1.0/docs/devguide/local-dom#observe-nodes)?